src/pages/ScriptList/index.tsx

```diff
@@ -132,6 +132,13 @@ const ScriptList = () => {
                       </a>
                     ) : null,
                   )}
+                  <a
+                    href={`/script/${script!.codeHashes[0]}/${script!.hashType}`}
```

Comment:
You shouldn't use `!`, you can put assert on it, like:
```
const script = scriptDataList.find(s => s.tag === label)
assert(script)
```
or you can render it with conditions, like:
```
{script && <a ...>...</a>}
```